node/src/Commands.ts

```diff
@@ -1584,6 +1584,56 @@ export function createBLPop(
     return createCommand(RequestType.BLPop, args);
 }
 
+/**
+ * @internal
+ */
+export function createFCall(
+    func: string,
+    keys?: string[],
+    args?: string[],
+): command_request.Command {
+    const params: string[] = [];
+    params.push(func);
+
+    if (keys !== undefined) {
```

Comment:
Can you simplify the checks?
```suggestion
    if (keys) {
```